src/controllers/user.controller/updateAnUser/queries/queries.js

```diff
@@ -87,8 +87,33 @@ const checkPhoneNumber = `query checkPhoneNumber($phoneNumber: String) {
   }
 }`;
 
+const getUserIdFromCognito = `query getUserIdFromCognito($cognito_sub: String) {
+  user(where: {cognito_sub: {_eq: $cognito_sub}}) {
+    id
+  }
+}`;
+
+const getUserActivityDetails = `query getUserActivities($cognitoSub: String) {
+  user_activities(where: { user:{
+    cognito_sub:{
+      _eq: $cognitoSub
+    }
```

Comment:
please add a filter here to get only the required 4 activities. If not, you could be fetching 1000s of activities & manually filtering in javascript unnecessarily.
`
where: { user: {}, activities: { identifier: { _in: [bio, website, github, skill] } } }
`